clang/lib/Serialization/ASTReader.cpp

```diff
@@ -4082,14 +4069,14 @@ void ASTReader::ReadModuleOffsetMap(ModuleFile &F) const {
                           : ModuleMgr.lookupByFileName(Name));
     if (!OM) {
       std::string Msg =
-          "SourceLocation remap refers to unknown module, cannot find ";
+          "cannot find module ";
```

Comment:
Would be good to keep some context in the error message, maybe remove just the `"SourceLocation"` part and leave the rest as it was?

Reply:
Done